app/models/item.rb

```diff
@@ -14,7 +14,7 @@ class Item <ApplicationRecord
   validates_numericality_of :inventory, greater_than_or_equal_to: 0
   
   def set_defaults
-    if self.image == ""
+    if self.image == "" || self.image.nil?
```

Comment:
if an image is created not on the websites form, the if statement doesn't handle that, so added a check for nil clause